cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/BuildCommandTest.java

```diff
@@ -872,6 +872,27 @@ public void testBuildBalProjectWithDumpRawGraphsFlag() throws IOException {
         ProjectUtils.deleteDirectory(projectPath.resolve("target"));
     }
 
+    @Test(description = "Build a package with corrupted Dependencies.toml file")
+    public void testBuildWithCorruptedDependenciesToml() throws IOException {
+        Path projectPath = this.testResources.resolve("corrupted-dependecies-toml-file");
+        cleanTarget(projectPath);
+        Path sourcePath = projectPath.resolve("Dependencies-corrupt.toml");
+        Path destinationPath = projectPath.resolve("Dependencies.toml");
+        Files.copy(sourcePath, destinationPath);
+        BuildCommand buildCommand = new BuildCommand(projectPath, printStream, printStream, false);
+        new CommandLine(buildCommand);
+        buildCommand.execute();
+        String buildLog = readOutput(true);
+        Assert.assertTrue(buildLog.contains("WARNING [Dependencies.toml:(6:1,18:1)] " +
```

Comment:
Can we check for an exact match here with the expected log written in a file? AFAIR we did this for most of the existing tests.